lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -15,10 +15,11 @@ def initialize(serializer, options = {})
           end
         end
 
-        def serializable_hash(options = {})
+        def serializable_hash(options = nil)
+          options = {}
```

Comment:
Thanks @bacarini for finding this bug in #1041, should be `options ||= {}`
:person_frowning: on me